src/types.ts

```diff
@@ -3,26 +3,29 @@ import type { BuiltinPresetName } from './presets'
 export type ModuleId = string
 export type ImportName = string
 
-export interface Import {
-  /** import name to be imported */
-  name: ImportName
-  /** Import as this name */
-  as?: ImportName
+export interface ImportCommon {
   /** Module specifier to import from */
   from: ModuleId
   /**
    * Priority of the import, if multiple imports have the same name, the one with the highest priority will be used
    * @default 1
    */
   priority?: number
+  /** If this import is disabled */
```

Comment:
```suggestion
  /** Disable auto import or preset */
```

Note this comment can be also misleading for other fields such as `priority` in which case i think duplication of few props makes more sense than extending like this for sake for documentation at least.